elm-package.json

```diff
@@ -1,7 +1,7 @@
 {
-    "version": "1.1.0",
+    "version": "1.0.0",
     "summary": "unlimited size integers",
-    "repository": "https://github.com/gilbertkennen/bigint.git",
+    "repository": "https://github.com/hickscorp/elm-bigint.git",
```

Comment:
?

Reply:
@zwilias I needed to be able to use BigInt right away, and as I didn't know whether or not the repo was active, I decided to publish it on my own. Somehow, I end-up with a compilation error whenever I try to use it anyways, so I de-prioritized this and rolled my own in the project's local tree. I'll fix this a bit later.